plugin/src/utils/gql.js

```diff
@@ -1,3 +0,0 @@
-const gql = ([string]) => string
-
-export default gql
```

Comment:
Ah nice you did delete it, glad to be rid of this :D

Reply:
yes i wasnt sure what it was? sometimes folks do this because the LSP server only offers language support for `gql` tag right now. hoping to find the free time to fix that soon!